hl13/code/modules/projectiles/guns/ballistic/ar2.dm

```diff
@@ -0,0 +1,29 @@
+/obj/item/gun/ballistic/automatic/hl13/ar2
```

Comment:
instead of having a different file for each weapon that needs to be loaded and increases compile time,  just put everything under some "same category file" so since the AR2 and the mp7 are both automatic, put them inside an automatic.dm file

Reply:
All organized now